sequencer/src/lib.rs

```diff
@@ -420,7 +420,9 @@ pub mod testing {
         (
             <SimpleBuilderImplementation as TestBuilderImplementation<SeqTypes>>::start(
                 NUM_NODES,
-                url.clone(),
+                format!("http://0.0.0.0:{port}")
+                    .parse()
+                    .expect("Failed to parse builder listener"),
```

Comment:
This is the listener. `espresso-dev-node` needs this when it is running in a docker container